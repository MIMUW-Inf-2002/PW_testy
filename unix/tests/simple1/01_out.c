#include <unistd.h>
#include <stdio.h>

#define SCHEDULE_N 10

const int period = 100000;

int main() {
    printf("Not the last line.\n");
    printf("Last line.\n");
    return 0;
}