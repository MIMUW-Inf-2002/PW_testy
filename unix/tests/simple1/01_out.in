run ./01_out
run ./01_out
run ./01_out
run ./01_out
run ./01_out
sleep 300
out 0
out 1
out 2
out 3
out 4
out 0
out 1
out 2
out 3
out 4
