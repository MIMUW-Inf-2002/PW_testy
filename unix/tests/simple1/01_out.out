======
Task 0 started: pid 47818.
Task 1 started: pid 47820.
Task 2 started: pid 47824.
Task 3 started: pid 47828.
Task 0 ended: status 0.
Task 4 started: pid 47834.
Task 1 ended: status 0.
Task 2 ended: status 0.
Task 3 ended: status 0.
Task 4 ended: status 0.
======
Task 0 stdout: 'Last line.'.
Task 1 stdout: 'Last line.'.
Task 2 stdout: 'Last line.'.
Task 3 stdout: 'Last line.'.
Task 4 stdout: 'Last line.'.
Task 0 stdout: 'Last line.'.
Task 1 stdout: 'Last line.'.
Task 2 stdout: 'Last line.'.
Task 3 stdout: 'Last line.'.
Task 4 stdout: 'Last line.'.
